pkgs/by-name/fl/flashprog/package.nix

```diff
@@ -60,6 +61,11 @@ stdenv.mkDerivation (finalAttrs: {
     install -Dm644 util/50-flashprog.rules "$out/lib/udev/rules.d/50-flashprog.rules"
   '';
 
+  passthru.updateScript = gitUpdater {
```

Comment:
https://github.com/NixOS/nixpkgs/pull/383218

This first pass is removing gitUpdater without arguments. But it seems that the proposal is to also replace ones that take parameters. It might be something worth considering?

Reply:
As far as I can tell, gitUpdate and nix-update-script are two very different things. nix-update-script failed for me in various cases while gitUpdate does the job perfectly fine.

Reply:
I don't see why the use of gitUpdater should be discouraged as long as there is no clear plan, proposal on the advantages and why this is needed. So I'm marking this as resolved.